### DISCUSSION TITLE
Github copilot not working

### DISCUSSION THREAD

**Discussion by HarshParmar25** [community/community#7716] [Copilot Conversations] [answered]
I'm unable to use github copilot whenever i start it , tit always shows this messsage

GitHub Copilot could not connect to server. Extension activation failed: "Failed to get copilot token"

**Comment by davecheney** [comment, marked as answer]
Hello, I'm sorry you are having difficulties. We've found that the copilot icon may not show up if copilot is not able to contact our servers. Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

**Reply by bi305** [reply]
GitHub Copilot could not connect to server. Extension activation failed: "Failed to get copilot token"
 i  am also  facing  the  same  issue

**Reply by free2wi** [reply]
I am behind firewall, vpn and many more network securities yes. What are your urls to whitelist for Copilot to work properly?

**Reply by Bob4F** [reply]
> Hello, I'm sorry you are having difficulties. We've found that the copilot icon may not show up if copilot is not able to contact our servers. Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

Hi I've been on the whitelist waiting list, can I get a whitelist please?
